test/e2e-v2/cases/cluster/zk/es/docker-compose.yml

```diff
@@ -47,9 +47,16 @@ services:
     extends:
       file: ../../../../script/docker-compose/base-compose.yml
       service: oap
+    volumes:
+      - ../../test-downsampling.yaml:/skywalking/config/meter-analyzer-config/test-downsampling.yaml
     environment:
       SW_CLUSTER: zookeeper
       SW_STORAGE: elasticsearch
+      SW_CORE_PERSISTENT_PERIOD: 5
+      SW_METER_ANALYZER_ACTIVE_FILES: test-downsampling
+      SW_CLUSTER_ZK_HOST_PORT: zk:2181
+      SW_CLUSTER_INTERNAL_COM_HOST: oap1
+      SW_CLUSTER_INTERNAL_COM_PORT: 11800
```

Comment:
Please increase TTL period to a very long time to make sure no TTL will delete the data as you hard coded the time bucket

> dataKeeperExecutePeriod: ${SW_CORE_DATA_KEEPER_EXECUTE_PERIOD:5} # How often the data keeper executor runs periodically, unit is minute

Reply:
done